CHANGELOG.md

```diff
@@ -48,6 +48,14 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 #### `Flex` Component
 - `wrap` prop is no longer accepted on `Flex`
 
+#### No longer available components
+- `CountdownButton` (only used by Toast)
+- `DecoratedInput` (use TextInput instead)
+- `IconInput` (use TextInput instead)
+
+#### `CountdownButton` Component removed
```

Comment:
you can just remove this section; the "removed components" section above should cover it